examples/3d/spotlight.rs

```diff
@@ -111,6 +121,19 @@ fn setup(
         transform: Transform::from_xyz(-4.0, 5.0, 10.0).looking_at(Vec3::ZERO, Vec3::Y),
         ..default()
     });
+
+    commands.spawn(TextBundle::from_section(
+        "\
+Controls
+--------
+Horizontal Movement: WASD
+Vertical Movement: Space and Shift
+Rotate Camera: Left and Right Arrows",
+        TextStyle {
+            font_size: 24.0,
```

Comment:
There an ongoing but somewhat lazy effort to standardize the example instructions with
- 12px top/left padding
- 20px font size

See <https://github.com/bevyengine/bevy/pull/8478>

Reply:
Fixed in c0e5ff8334b109a505d37cdac1153a580b972aa2, thanks!